src/drive/actions/async.js

```diff
@@ -126,8 +127,8 @@ class Stack {
 }
 
 class PouchDB {
-  constructor({ folder, recent, sort }) {
-    this.folderIndex = folder
+  constructor({ folders, recent, sort }) {
+    this.folderIndex = folders
```

Comment:
That was a typo, the index is actually called `folders`. Seemed to work without it though ?